web/content/2.5.md

```diff
@@ -306,15 +306,28 @@ Puppet 5 is officially End of Life. Foreman {{page.version}} still supports Pupp
 
 ### Katello headline features
 
-#### Features
+For the full list of changes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-4.1/CHANGELOG.md).
 
-#### Deprecations
+## Features
+* New `hammer content-export` and `hammer content-import` workflows based on Pulp 3
+  - Auto-import custom repos
+  - Imports and exports now show up in Audits
+  - Automatically create content view on import
+  - Import/export now supports File and Ansible repo types
+  - `hammer content-import list` command now available
+* Improved handling of Activation Keys and other Katello-related items for Global Host Registration feature
+* More updates to new Content View preview UI at /labs/content_views
+* Upgraded to pulpcore 3.11
+* Added global_download_concurrency setting (defaults to 5)
+* Katello now supports logging correlation_id for pulp3 operations to make it easier to debug across different log files
 
-#### Notes
 
-### Bug Fixes
+## Deprecations
+* API endpoints for katello-agent actions have been deprecated. Consider switching to remote execution.
+* New deprecation warning on the Register Content Host page - Use Global Registration instead (/hosts/register)
 
-### Contributors
+
+### Contributors to Foreman 2.5
 * Adam Ruzicka
 * Adi Abramovich
 * Adi Abramovitch
```

Comment:
Are there two people?

Reply:
Probably not ;)  @adiabramovitch Which is the correct spelling?

Reply:
Adi Abramovich (: